MobileCenterAnalytics/MobileCenterAnalytics/Support/MobileCenterAnalytics.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework UIKit -framework CoreTelephony;
```

Comment:
Why do all modules need the sqlite dependency?
This should only be necessary for core, right?!